azure-pipelines.yml

```diff
@@ -125,7 +125,7 @@ jobs:
     inputs:
       command: test
       projects: '**/*.ITest/*.csproj'
-      arguments: '--blame --no-build -c $(buildConfiguration) -maxcpucount:1 /p:CopyLocalLockFileAssemblies=true $(skipFilter) --collect:"XPlat Code Coverage" --settings coverlet.runsettings --logger trx --results-directory $(Build.SourcesDirectory)'
```

Comment:
Removing `--no-build` seems to slow down test execution by enough time that the config server is ready for requests by the time the tests run (2 of the failing tests seemed to be for this reason), but it isn't the most scientific way to make sure the config server is ready